vllm/utils.py

```diff
@@ -922,3 +922,35 @@ def parse_args(self, args=None, namespace=None):
                 processed_args.append(arg)
 
         return super().parse_args(processed_args, namespace)
+
+
+class DeferredTensor:
```

Comment:
Why isn't it possible to subclass/use Meta tensor for example?

Reply:
Meta tensor is used to respond metadata related query now.